src/command_modules/azure-cli-storage/setup.py

```diff
@@ -30,7 +30,7 @@
 ]
 
 DEPENDENCIES = [
-    'azure-multiapi-storage==0.1.3',
+    'azure-multiapi-storage==0.1.4',
```

Comment:
Why upgrade the dependency here? Is it related?

Reply:
Yes it is related.
I released 0.1.4 as the sdist for 0.1.3 had a missing __init__.py file.
Since Homebrew requires building from source, I had to release 0.1.4.

https://github.com/Azure/azure-multiapi-storage-python#014